ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandRunnerTest.java

```diff
@@ -17,34 +17,41 @@
 package io.confluent.ksql.rest.server.computation;
 
 import org.apache.kafka.clients.consumer.KafkaConsumer;
-import org.apache.kafka.clients.producer.KafkaProducer;
 import org.apache.kafka.common.serialization.Deserializer;
-import org.apache.kafka.common.serialization.Serializer;
+import org.junit.After;
+import org.junit.AfterClass;
 import org.junit.Assert;
+import org.junit.ClassRule;
 import org.junit.Test;
 
 import java.util.HashMap;
 import java.util.Map;
 
 import io.confluent.kafka.serializers.KafkaJsonDeserializer;
-import io.confluent.kafka.serializers.KafkaJsonSerializer;
 import io.confluent.ksql.metastore.MetaStoreImpl;
 import io.confluent.ksql.rest.entity.CommandStatus;
 import io.confluent.ksql.rest.server.StatementParser;
 import io.confluent.ksql.rest.server.mock.MockCommandStore;
 import io.confluent.ksql.rest.server.mock.MockKafkaTopicClient;
 import io.confluent.ksql.rest.server.mock.MockKsqkEngine;
-import io.confluent.ksql.rest.server.utils.TestUtils;
+import io.confluent.ksql.testutils.EmbeddedSingleNodeKafkaCluster;
+import io.confluent.ksql.util.KsqlConfig;
 
 public class CommandRunnerTest {
 
+  @ClassRule
+  public static final EmbeddedSingleNodeKafkaCluster CLUSTER = new EmbeddedSingleNodeKafkaCluster();
```

Comment:
Why does this need an embedded broker? Could we not just use the `MockKafkaConsumer` etc?

Reply:
Refactored this test to mock requires objects instead of using EmbeddedSingleNodeKafkaCluster.